README.md

```diff
@@ -31,11 +31,16 @@ Run with: mvn verify
                             </goals>
                             <configuration>
                                 <projectName>cucumber-jvm-example</projectName>
+                                <!-- output directory for the generated report -->
                                 <outputDirectory>${project.build.directory}</outputDirectory>
+                                <!-- optional, defaults to outputDirectory if not specified -->
+                                <inputDirectory>${project.build.directory}/jsonReports</inputDirectory>
```

Comment:
nice, so then can you now set the directory different than outputDirectory and different from jsnoReports?

Reply:
Yes, that's the idea.